service/dap/server_test.go

```diff
@@ -4291,64 +4290,66 @@ func TestNextParked(t *testing.T) {
 						}
 					}
 				},
-				disconnect: false,
+				terminated: true,
```

Comment:
Will this work? The harness only looks at terminated if disconnect is true.

Reply:
will add disconnect: true just in case.